### PR TITLE
Adds Explore your APIs

### PR DESCRIPTION
**Pull request by mnaumanali94**

### CODE REVIEW

File: docs/2.-governance/b.exploring-your-api-projects.md [hunk 1]
```diff
@@ -3,3 +3,55 @@ tags: [Governance]
...
+
+With all your API artifacts in a single place, you can use this to:
+
+### Find obscure APIs
+*You can't fix what you can't see.* APIs that have long become lost in the organization suddenly become available for developers to use and champions to govern, which helps avoid duplications and remove inconsistencies. Technical writers can improve documentation, API teams can write specs for these APIs which can help your business add value by using what is already available. 
+
+### Promote consistency
+When API designers can find existing models and example styles, they build more consistent interfaces. Engineers integrate more quickly, but also they'll have the information to design thoughtful APIs from the start. Technical writers can discover documentation for dor references. API designers can spot inconsistencies in their design and work towards making them consistent. 
+
+With a good view of your current state of design and documentation, you can create custom style guides to enforce consistency without drilling down too hard on the current practices.
```
Comment (philsturgeon) [inline docs/2.-governance/b.exploring-your-api-projects.md:40]: We should be putting in loads of links to various bits. When we talk about custom style guides we can link to the style guides guide, all to make things easy and stop people needing to look around if they're not sure what something means.

File: docs/2.-governance/b.exploring-your-api-projects.md [hunk 2]
```diff
@@ -3,3 +3,58 @@ tags: [Governance]
 ---
 
 # Exploring your API projects
+
+All APIs in your organization, whether internal or external, provide potential business value in terms of new use cases, increased productivity, or visibility. Thus, it is essential to manage all of them.
```
Comment (philsturgeon) [inline docs/2.-governance/b.exploring-your-api-projects.md:7]: What does visibility mean in this context?

File: docs/2.-governance/b.exploring-your-api-projects.md [hunk 3]
```diff
@@ -3,3 +3,58 @@ tags: [Governance]
 ---
 
 # Exploring your API projects
+
+All APIs in your organization, whether internal or external, provide potential business value in terms of new use cases, increased productivity, or visibility. Thus, it is essential to manage all of them. 
+
+Organizations with a growing API stack and team, sometimes being developed in different parts of the globe, lose sight of what's already available and its current state, which leads to inconsistencies and duplication that cost the business dearly. Moreover, the cost of adding a new API into your stack increases exponentially.
```
Comment (philsturgeon) [inline docs/2.-governance/b.exploring-your-api-projects.md:9]:
```suggestion
Organizations with a growing API stack and team, sometimes being developed in different parts of the globe, it's possible to lose sight of what's already available and its current state. Poor visibility into the currently available APIs leads to inconsistencies and duplication that cost the business dearly. Moreover, the cost of adding a new API into your stack increases exponentially. 
```

File: docs/2.-governance/b.exploring-your-api-projects.md (hunk 2 above)
Comment (mnaumanali94) [reply docs/2.-governance/b.exploring-your-api-projects.md:7]: Visibility in terms of teams exposing their data which might normally live in silos. I'll make it more clear in the next cycle.

**Pull request merged**